tests/test_architecture.py

```diff
@@ -11,6 +11,22 @@
 GNAME = "framework-testing"
 
 
+def summary_exception(values: np.ndarray,
```

Comment:
Since you are maintaining generic methods for testing in MFETestClass, it would also be interesting to put this function in MFETestClass.
is there a constraint to don't put this function in MFETestClass?

Reply:
Hi Edesio. There's no real constraint for this. I choose to don't implement those functions as MFETestClass methods to simulate as much as possible the real functioning of the pymfe library, given that the summary functions are not methods of any class.